icu4c/source/data/translit/Hebr_Latn.txt

```diff
@@ -24,7 +24,7 @@
 # Paste in hebrew text in Input, and hit Transliterate.
 #
 # For more information, see:
-#   https://unicode-org.github.io/icu/userguide/transforms/general/
+#   http://icu.sourceforge.net/userguide/Transform.html
```

Comment:
@AEApple @DraganBesevic @macchiati this is a regression. SourceForge stuff is ancient. I am baffled that someone dug up a URL from the stone age.

Please revert this change in CLDR.

Reply:
to be continued via email...